pyproject.toml

```diff
@@ -0,0 +1,93 @@
+[build-system]
+requires = [ "setuptools>=61.1" ]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "scapy"
+dynamic = [ "version", "readme" ]
+authors = [
+    { name="Philippe BIONDI" },
```

Comment:
Note: I removed phil's email because the tools refused "phil(at)secdev.org". Thoughts? (we can add emails to maintainers now if we want to, but brace the spam)

Reply:
I believe that twine will complain when pushing Scapy to PyPi. What was the error ?

Reply:
It's super fancy error btw
![image](https://user-images.githubusercontent.com/10530980/215900306-1a79dcfc-7399-48dd-9007-a685f9136adc.png)